src/modules/core/components/ItemsList/ItemsList.tsx

```diff
@@ -232,15 +248,23 @@ class ItemsList extends Component<Props, State> {
               parseInt(styles.paddingValue, 10)}px`,
           }}
         >
-          <button
-            disabled={disabled}
-            type="button"
-            className={id < 0 ? styles.itemHeading : styles.item}
-            onClick={() => this.handleSelectItem(id)}
-            title={decoratedName}
-          >
-            {decoratedName}
-          </button>
+          <Tooltip content={tooltipContent} placement="bottom">
+            {/*
+             * Must use a `div` here, as `mouseleave` event isn't fired on buttons that are `disabled`.
+             * See: https://github.com/facebook/react/issues/4251
+             */}
+            <div>
```

Comment:
As an optional improvement _(again, optional, so up to you...)_ we can make this `div` element be displayed as `inline-block`, so that the tooltip appears centered to the actual domain name:

![Screenshot from 2019-11-08 14-57-21](https://user-images.githubusercontent.com/1193222/68478346-4c50a880-0238-11ea-8be5-69cb03204278.png)